aula 01.py

```diff
@@ -1,4 +1,4 @@
-nome = input("whats your name? ")
-idade = input("whats your age? ")
-print("seu nome é ",nome,
-      "e sua idade é ",idade,". ")
\ No newline at end of file
+nome = input("Qual o seu nome? ")
+idade = input("Qual é sua idade? ")
+
+print("Seu nome é",nome,"e sua idade é",idade,". ")
```

Comment:
Poderia ser com um espaço antes da idade
```suggestion
print("Seu nome é", nome, "e sua idade é", idade, ". ")
```